### PR TITLE
Hide avoid standby mode feature on *nux OS

### PR DESCRIPTION
**Pull request by ghubstan**
Displays standby mode button on only on Windows and OSX,
and hides it on Linux and Unix distributions.

Formatted source file, resulting in rearranged class
level field declarations.

Fix for Issue #3223

### CODE REVIEW

**Review by chimp1984 — CHANGES_REQUESTED** [review]
Missing adjustment of num rows.
`TitledGroupBg titledGroupBg = addTitledGroupBg(root, gridRow, 8, Res.get("setting.preferences.general"));` -> 8 need to be 7 in case its not displayed

Is it confirmed that on Unix there is no need for the hack with playing the sound to avoid standby?

**Comment by ghubstan** [comment]
@chimp1984, the row adjustment fix has been committed.

I have not confirmed there is no need for the Unix sound hack. Can @wiz or @niyid confirm there is no need for the Unix sound hack?

File: desktop/src/main/java/bisq/desktop/main/settings/preferences/PreferencesView.java [hunk 1]
```diff
@@ -97,14 +98,22 @@
...
+    private final Preferences preferences;
+    private final FeeService feeService;
+    //private final ReferralIdService referralIdService;
+    private final AssetService assetService;
+    private final FilterManager filterManager;
+    //private ComboBox<BaseCurrencyNetwork> selectBaseCurrencyNetworkComboBox;
+    private final DaoFacade daoFacade;
+    private final BSFormatter formatter;
+    private final boolean daoOptionsSet;
+    private final boolean displayStandbyModeFeature;
```
Comment (freimair) [inline desktop/src/main/java/bisq/desktop/main/settings/preferences/PreferencesView.java:110]: please do not commit non-changes. makes it hard to review and introduces unnecessary big change sets.

**Review by freimair — CHANGES_REQUESTED** [review]

File: desktop/src/main/java/bisq/desktop/main/settings/preferences/PreferencesView.java (hunk 1 above)
Comment (ghubstan) [reply desktop/src/main/java/bisq/desktop/main/settings/preferences/PreferencesView.java:110]: I don't understand.   Are the non-changes you mentioned references to the comments (lines 103, 106)?  Those were there from previous commits (before this PR).  I did not add these comments.  Should I delete them?

Or, are you referring to the Intellij code reformatting which rearranged the order of field definitions?  I have been reformatting files I touch, using the IDE code style versioned in git.  Should I not do that?

Comment (freimair) [reply desktop/src/main/java/bisq/desktop/main/settings/preferences/PreferencesView.java:110]: > Or, are you referring to the Intellij code reformatting which rearranged the order of field definitions? I have been reformatting files I touch, using the IDE code style versioned in git. Should I not do that?

that is what I meant. please only commit changes that have something to do with the topic of your PR. Thanks!

**Pull request closed without merging**